packages/support/resources/lang/ja/actions/attach.php

```diff
@@ -4,36 +4,36 @@
 
     'single' => [
 
-        'label' => 'Attach',
+        'label' => '追加',
 
         'modal' => [
 
-            'heading' => 'Attach :label',
+            'heading' => ':labelの追加',
```

Comment:
I don't know Japanese, but shouldn't there be a space between the label and the rest of the string?